src/client.ts

```diff
@@ -347,19 +364,23 @@ export class ClobClient {
     public async createOrder(userOrder: UserOrder): Promise<SignedOrder> {
         this.canL1Auth();
 
-        return this.orderBuilder.buildOrder(userOrder);
+        const tickSize = await this.getTickSize(userOrder.tokenID);
```

Comment:
I think we should allow this to be passed as a param in which case we don't make the api call. The FE will likely already have the tick size available at order creation time.

Reply:
New parameter added ✅